### DISCUSSION TITLE
Seems that when mkdocs is the source for an iframe, it thinks the page width is too small

### DISCUSSION THREAD

**Discussion by githubjbw** [squidfunk/mkdocs-material#4501] [Q&A]
I'm trying to include a mkdocs url as a source for an iframe, which works but there is no left nav and instead the drop down nav is present as if the page width is too small. This happens even though the iframe width is sufficiently wide enough. Not sure if the iframe width is not known to mkdocs but it seems that the left nav should be shown. The iframe width is 1280px wide.

Any thoughts?

This is what I'm trying:

```
<div class="resp-container">
  <iframe
    class="resp-iframe"
    id="docs-iframe"
    src="https://mkdocs-site-url-here/"
    allowfullscreen
    style="border:none"
    sandbox="allow-scripts">
  </iframe>
</dev>
```

CSS
```
.resp-container {
    position: relative;
    overflow: hidden;
    padding-top: 56.25%;
	max-width: 100%;
	width: 100%;
}
.resp-iframe {
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    border: 0;
}
```

**Comment by squidfunk** [comment]
Thanks for asking. Honestly, this is beyond the scope of this project. What you're writing makes sense but I would need to debug what you describe as well, and this won't benefit many users since it's an edge case. You might have more luck on StackOverflow.

**Comment by githubjbw** [comment]
Thanks,

I figured out a way to make it work, by adding in padding-left and padding-right to the parent class for the iframe. I'm not 100% convinced this is really best way but it seems to work in lieu of any other way of making it work right now.

```
.resp-container {
    position: relative;
    left: 0;
    right: 0;
    height 0;
    width: 100%;
    padding-top: 600px;
    padding-left: 0px;
    padding-right: 0px;
}

.resp-iframe {
    position: absolute;
    top: 0;
    left: 0;
    bottom 0;
    right 0;
    width: 100%;
    height: 100%;
    border: 0;
}
```
